projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/ExperimentalGroup.java

```diff
@@ -84,7 +84,13 @@ public int sampleSize() {
     return this.sampleSize;
   }
 
-  public List<BiologicalReplicate> biologicalReplicates() { return this.biologicalReplicates; }
+  public Long experimentalGroupId() {
+    return this.experimentalGroupId;
+  }
+
+  public List<BiologicalReplicate> biologicalReplicates() {
+    return this.biologicalReplicates;
+  }
```

Comment:
this methods must not return the reference to the original list, which can then be manipulated with side effects by the client.

please return a shallow copy.